docs/sources/flow/getting-started/migrating-from-prometheus.md

```diff
@@ -0,0 +1,184 @@
+	job_name       = "prometheus"
+	scrape_timeout = "45s"
+}
+
+prometheus.remote_write "default" {
+	endpoint {
+		name = "grafana-cloud"
+		url  = "https://prometheus-us-central1.grafana.net/api/prom/push"
+
+		basic_auth {
+			username = "USERNAME"
+			password = "PASSWORD"
+		}
+
+		queue_config {
+			capacity             = 2500
+			max_shards           = 200
+			max_samples_per_send = 500
+		}
+
+		metadata_config {
+			max_samples_per_send = 500
+		}
+	}
+}
+```
+
+### Debugging
+
+1. If the convert command cannot convert a Prometheus configuration,
+   diagnostic information is sent to `stderr`. You can bypass
+   any non-critical issues and output the flow configuration using a 
+   best-effort conversion by including the `--bypass-errors` flag.
+   
+    > Be aware that the behavior may not match when bypassing errors.
+
+    ```bash
+    grafana-agent convert --format=prometheus --bypass-errors --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+    ```
+
+1. You can also output a diagnostic report by including the `--report` flag.
+
+    ```bash
+    grafana-agent convert --format=prometheus --report=OUTPUT_REPORT_PATH --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+    ```
+
+    * Replace `OUTPUT_REPORT_PATH` with the output path for the report.
+
+    Using the example Prometheus configuration from above the diagnostic report provides the following information:
+
+    ```
+    (Info) Converted scrape_configs job_name "prometheus" into...
+      A prometheus.scrape.prometheus component
+    (Info) Converted 1 remote_write[s] "grafana-cloud" into...
+      A prometheus.remote_write.default component
+    ```
+
+## Run a Prometheus configuration
+
+If you’re not ready to switch to a flow configuration, you can also run your
+existing Prometheus configuration without saving it as a flow configuration.
+This allows you to try flow mode without modifying your existing
+Prometheus configuration infrastructure.
+
+> In this task, we will use the [run][] CLI command to run Grafana Agent in flow
+> mode using a Prometheus configuration.
+
+[Start the agent][] in flow mode and include the command line flag
+   `--config.format=prometheus`. Your configuration file must be a valid
+   Prometheus configuration file rather than a flow mode configuration file.
+
+
+### Debugging
+
+1. You can follow the convert CLI command [debugging][] instructions to
+   generate a diagnostic report.
+
+1. Refer to the Grafana Agent [Flow Debugging][] for more information about a running Grafana
+   Agent in flow mode.
+
+3. If the Prometheus configuration provided cannot be converted,
+   diagnostic information is sent to `stderr`. You can bypass
+   any non-critical issues and start the Agent by including the
+   `--config.bypass-conversion-errors` flag in addition to
+   `--config.format=prometheus`.
+
+    > Be aware that the behavior may not match when bypassing errors
+    > and doing so should be avoided in Production systems.
```

Comment:
```suggestion
    {{% admonition type="caution" %}}
    If you bypass the errors, the behavior of the converted configuration may not match the original configuration. 
    Make sure you fully test the converted configuration before using it in a production environment.
    {{% /admonition %}}
```

Is this a strong enough statement? Or should we go with `Do not use this flag in production environments.`